src/main/java/uk/ac/ebi/eva/commons/models/converters/data/DBObjectToSamplesConverter.java

```diff
@@ -0,0 +1,71 @@
+package uk.ac.ebi.eva.commons.models.converters.data;
+
+import com.mongodb.BasicDBObject;
+import com.mongodb.DBObject;
+import org.opencb.biodata.models.feature.Genotype;
+import org.opencb.datastore.core.ComplexTypeConverter;
+
+import uk.ac.ebi.eva.commons.models.data.VariantSourceEntry;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Simplified samples converter that does not require the names of the samples, as it will compress them in mongo.
+ */
+public class DBObjectToSamplesConverter implements ComplexTypeConverter<VariantSourceEntry, DBObject> {
```

Comment:
agreed to leave only the write part of these converters. the read part will be done directly in the WS or variation commons